doc/adoc/user_guide.adoc

```diff
@@ -157,11 +157,19 @@ manually. Once done, the upgrade process uses `zypper dup` and expects
 all required repositories to be setup correctly.
 
 Migration product target::
-If set, the migration will use that product as a target.
+The default product target is SLES-15-SP1. The migration_product setting can be used
+to change the product target. 
+
+[NOTE]
+Changing the product target is not officially supported
+
+If migration_product is set, the migration will use that product as a target.
 The product must be specified with the triplet PROD_NAME/VERSION/ARCH, for example:
 
 [listing]
-migration_product: 'SLES/15/x86_64'
+migration_product: 'SLES/15.1/x86_64'
+
+would set the product target to SLES-15-SP1
```

Comment:
We can state the the product tripplet is composed of the information from `/etc/products.d/baseproduct` by combining the `name`, `version`, and `arch` values from the XML document.